### DISCUSSION TITLE
Bug in iOS client!

### DISCUSSION THREAD

**Discussion by ExploitMann** [community/community#58308] [Mobile] [answered]
Example on [this video](https://youtube.com/shorts/uQWHfXXjKMI?feature=share)
Please fix this!
Device: iPhone 7
iOS: 15.7.6 (latest)

**Comment by iskae** [comment, marked as answer]
Hi @secure-fun 👋 

Thanks for reaching out and including a video too 🎉 

We already have an internal tracking issue for this and I will be adding your report in there too 👍 

Please stay tuned for the future versions for a potential fix 🙇 

Cheers!

**Comment by Iron-Ham** [comment]
Hi @secure-fun, we have a fix rolling out to address this. If you're still experiencing this in ~24 hours -- please let me know. 


Thanks,
Hesham
